uber/site_sections/summary.py

```diff
@@ -52,8 +53,12 @@ def index(self, session):
                 counts['interests'][c.INTERESTS[val]] += 1
             if a.paid == c.PAID_BY_GROUP and a.group:
                 counts['groups']['paid' if a.group.amount_paid else 'free'] += 1
-            if a.total_donation >= c.SUPPORTER_LEVEL:
-                counts['supporter_level'] += 1
+
+            donation_amounts = list(counts['donation_tiers'].keys())
+            for index, amount in enumerate(donation_amounts):
+                next_amount = donation_amounts[index + 1] if index + 1 < len(donation_amounts) else six.MAXSIZE
+                if a.total_donation >= amount and a.total_donation < next_amount:
```

Comment:
Interesting that the old code used `total_donation` as the property to check. My first thought was that we should only check `amount_extra` since that's what the actual kick-in levels correspond to, but on second thought, maybe we want both. We definitely had trouble the first year we introduced the `extra_donation` box with people using it thinking it like Kickstarter, where having a certain donation automatically qualified you for the corresponding kick-in levels.

It'd be nice if we had a list that was like:
> Number of attendees who've kicked in for (or donated up to):
> $XXX (Kick-in level name): 50 (67 including non-kick-in donations)
> $XXX (Kick-in level name): 32 (47 including non-kick-in donations)
> etc
>
> NOTE: These numbers are not cumulative, but we generally promise that all kick-in levels include the levels below them. If ordering swag, please take that into account.